taskfiles/releases.yml

```
@@ -3,50 +3,124 @@ version: '3'
             echo "Invalid usage."
             echo
             echo "Arguments:"
-            echo "<command> The Helmfile command to use. $([ -z "$COMMAND" ] && echo ❌ Not specified || echo ✅ Current value is $COMMAND)."
-            echo "<namespace> Must be a valid namespace. $([ -z "$NAMESPACE" ] && echo ❌ Not specified || echo Current value is $NAMESPACE. $([ ! -f "$NAMESPACE_FILE_PATH" ] && echo ❌ Expected file $NAMESPACE_FILE_PATH does not exist || echo Expected file ✅ $NAMESPACE_FILE_PATH exists))."
+            printf "<command> The Helmfile command to use. "
+            if [ -z "$COMMAND" ]; then
+                echo "❌ Not specified"
+            else
+                echo "✅ Current value is $COMMAND"
+            fi
+            
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)